spec/javascripts/packages/document-capture/components/selfie-capture-spec.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
```

Comment:
These tests aren't super thorough. It's made more difficult by the fact that JSDOM support of the features we're using is mostly non-existent, requiring quite a few custom mock implementations. It would be good to circle back here and flesh out the tests when time allows.